src/back/project/src/api/di-plugin-param.cpp

```diff
@@ -57,7 +58,9 @@ formats::json::Value DiPluginParam::Post(
 	const formats::json::Value& body,
 	formats::json::ValueBuilder& res) const
 {
-	const auto diPluginParam = body.As<model::DiPluginParam>();
+	auto diPluginParam = body.As<model::DiPluginParam>();
```

Comment:
Давай пожалуйста в типовом коде, который копируется как есть из места в место, создавать сущности с типовыми именами, когда работает только с одним типом сущности. Наличие уникального имени в 50 одинаковых файлах никакой пользы не несёт. А если имя будет типовым, то и копировать этот код будет проще.
```suggestion
	auto item = body.As<model::DiPluginParam>();
```

Reply:
Готово.